debian/rules

```diff
@@ -2,12 +2,14 @@
 # -*- makefile -*-
 
 export DH_OPTIONS
+PYTHON_BINARY := $(shell { command -v python3.4 || command -v python3.3 || command -v python2.7; } 2>/dev/null)
```

Comment:
My understanding of this is that python2.7 is required because that is what the virtualenv uses. We had a long thread about this internally in #operations and @solarkennedy was pretty adamant that 2.7 was necessary.  If it's not, this commit can be used: 3d552af0d47dd